app/bingey-ui/src/components/Watchlist/TitleCard/TitleCard.jsx

```diff
@@ -0,0 +1,45 @@
+import React, { useState } from 'react';
+import Card from '@material-ui/core/Card';
+import CardActionArea from '@material-ui/core/CardActionArea';
+import CardContent from '@material-ui/core/CardContent';
+import CardMedia from '@material-ui/core/CardMedia';
+import Typography from '@material-ui/core/Typography';
+
+import 'components/Watchlist/TitleCard/cardStyle.scss';
```

Comment:
As we decided to use relative imports for specific component styling files, this should be relative too

Reply:
o tai nesakem kad nuo components aplanko pradedam?